RELEASE-NOTES.txt

```diff
@@ -11,6 +11,7 @@
 * [**] [internal] Refactor fetching one single media item. [#21567]
 * [**] [internal] Refactored how the total number of WordPress Media is fetched. [#21568]
 * [**] [internal] Refactor fetching metadata of VideoPress videos. [#21569]
+* [*] Fix support for `.heic` uploads to self-hosted sites [#21622]
```

Comment:
I think there's a separate process to update release notes after code freeze

Reply:
ref: p77Llu-bGf-p2

Reply:
Removed for now thanks